config/environments/development.rb

```diff
@@ -65,4 +65,18 @@
     '130c1c3a95d040359d0e10ccc5ce83f4.vfs.cloud9.us-east-1.amazonaws.com'
   # Whitelist a test domain
   config.hosts << /application\.local\Z/
+
+  config.action_mailer.perform_deliveries = false
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3_000 }
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: 'smtp.gmail.com',
+    port: 587,
+    domain: 'gmail.com',
+    user_name: 'picachu.csce431@gmail.com',
+    password: 'Ourgroupproject431',
+    authentication: 'plain',
+    enable_starttls_auto: true
+  }
```

Comment:
This email address can no longer be considered secure, as the credentials are now publicly-available for all to see. We will need to delete this email address and create a new one.

Please never include any credentials in plaintext in any Git repository, as it could be made public in the future, exposing all of your information. Use environment variables for this purpose instead.

```suggestion
  config.action_mailer.smtp_settings = {
    address: 'smtp.gmail.com',
    port: 587,
    domain: 'gmail.com',
    user_name: ENV['MAILER_USERNAME'],
    password: ENV['MAILER_PASSWORD'],
    authentication: 'plain',
    enable_starttls_auto: true
  }
```